modules/customization/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -0,0 +1,74 @@
+[id='proc-customize-rhdh-sidebar-menuitems_{context}']
+= Customizing the sidebar navigation menu items for your {product-short} instance
+
+You can customize the sidebar menu items in your `app-config-rhdh.yaml` file. The order and parent-child relationships of sidebar main menu items can be customized using the `dynamicPlugins.frontend.default.main-menu-items.menuItems` field in the `app-config-rhdh.yaml` file:
+
+[source,yaml]
+----
+dynamicPlugins:
+  frontend:
+    <package_name>: # same as `scalprum.name` key in plugin's `package.json`
+      menuItems: # optional, allows you to configure plugin menu items in the main sidebar navigation
+        <menu_item_name>: # unique name in the plugin menu items list <1>
+          icon: fooIcon # optional, same as `menuItem.icon` in `dynamicRoutes` <2>
+          title: Foo Plugin Page # optional, same as `menuItem.text` in `dynamicRoutes` <3>
+          priority: 10 # optional, defines the order of menu items in the sidebar <4>
+          parent: favorites # optional, defines parent-child relationships for nested menu items <5>
+----
+
+<1> This attribute represents a unique name in the main sidebar navigation. It can denote either a standalone menu item or a parent menu item. If this attribute represents a plugin menu item, the name of the attribute must match with the corresponding path in `dynamicRoutes`. For example, if `dynamicRoutes` defines `path: /my-plugin`, then `menu_item_name` must be defined as `my-plugin`.
```

Comment:
Hi @hmanwani-rh , I have added some explanation to `path` and `<menu_item_name>` mapping logic, please refer this [link](https://github.com/janus-idp/backstage-showcase/pull/1623/files#diff-8d2e5712d5ef532a983602f4c771597e41d8c082d7801f59f6003fe9f66e2a89R528) for details.

Reply:
Done